example/src/KlaviyoReactWrapper.ts

```diff
@@ -97,7 +97,9 @@ export const setPushToken = async () => {
 
 export const setProfileAttribute = async () => {
   try {
-    Klaviyo.setProfileAttribute('CUSTOM', generateRandomName(12));
+    Klaviyo.setProfileAttribute(ProfileProperty.CITY, generateRandomName(5));
```

Comment:
added a couple properties outside of the custom one so that manual testing is easier.

Reply:
Nice.
Yeah I'd love to update this sample app someday to have an input form here instead of just generating random data.